### PR TITLE
Update for SIG tutorial 

### PR DESCRIPTION
**Pull request by slab14**
This updates the SIG tutorial document (remote_sig.md) to reflect the recent changes to scionlab. This was based on PR #134 & PR #123 and should fix the concerns with those PR (and Issue #100).

---
This change is [Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/144)

### CODE REVIEW

File: content/apps/remote_sig.md [hunk 1]
```diff
@@ -6,316 +6,273 @@ nav_order: 60
...
-
-The [SCION IP Gateway `SIG`](https://github.com/netsec-ethz/scion/tree/scionlab/go/sig) enables legacy IP applications to communicate over SCION. This tutorial describes how to set up two SIGs locally to test the SIG can enable any IP application to communicate over SCION.
+The [SCION IP Gateway `SIG`](https://github.com/netsec-ethz/scion/tree/scionlab/go/sig) enables legacy IP applications to communicate over SCION. This tutorial describes how to set up two SIGs locally to test the SIG can enable any IP application to communicate over SCION (this can be extended to support multiple SIG enpoints).
 
 ## Environment
 
 To test the SIG we will make use of the Vagrant configurations provided on [scionlab.org](https://scionlab.org/).
 Set up a Vagrant VM on a host A and a host B using the instructions from [Virtual machine with VPN](https://netsec-ethz.github.io/scion-tutorials/virtual_machine_setup/dynamic_ip/).
 
+Building the SIG binary requires go version 1.13. This is newer than many default packages, and can be installed as followed.
```
Comment (FR4NK-W) [inline content/apps/remote_sig.md]: I think it makes more sense to refer to the `building from source` docs page:
https://docs.scionlab.org/content/install/src.html
for building. The Go version is given by the go.mod file.
This also prevents people from ending up with duplicate entries in their .profile and makes it easier to update the build instructions.

**Review by FR4NK-W — CHANGES_REQUESTED** [review]
Thanks for the PR
Only minor comments

File: content/apps/remote_sig.md [hunk 2]
```diff
@@ -6,316 +6,273 @@ nav_order: 60
...
 ```shell
-export IA=$(cat $SC/gen/ia)
-export IAd=$(cat $SC/gen/ia | sed 's/_/\:/g')
-export AS=$(cat $SC/gen/ia | cut --fields=2 --delimiter="-")
-export ISD=$(cat $SC/gen/ia | cut --fields=1 --delimiter="-")
+# First, set some variables that will be used throughout this tutorial
+export SC=/etc/scion
+export LOG=/var/log/scion
+export ISD=$(ls /etc/scion/gen/ | grep ISD | awk -F 'ISD' '{ print $2 }')
+export AS=$( ls /etc/scion/gen/ISD${ISD}/ | grep AS | awk -F 'AS' '{ print $2 }')
```
Comment (FR4NK-W) [inline content/apps/remote_sig.md]: >AS=$( ls

Stray space

**Comment by slab14** [comment]
Thanks for the comments, updated to address.

**Review by slab14 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/144)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @FR4NK-W)

___
*[content/apps/remote_sig.md, line 16 at r1](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/144#-M4Uj_G9G_WCGTsCyGPZ-r1-16:-M4yvtJj9vvIIkQ29plN:b-7t22qw) ([raw file](https://github.com/netsec-ethz/scion-tutorials/blob/1f2a4db3a1e0d694cd0bcfc0db1cb55f0615bb47/content/apps/remote_sig.md#L16)):*
<details><summary><i>Previously, FR4NK-W wrote…</i></summary><blockquote>

I think it makes more sense to refer to the `building from source` docs page:
https://docs.scionlab.org/content/install/src.html
for building. The Go version is given by the go.mod file.
This also prevents people from ending up with duplicate entries in their .profile and makes it easier to update the build instructions.
</blockquote></details>

Good point. I removed the reference to a specific Go version (and download) and referenced the `build from source` docs to get the correct Go version. Done.
___
*[content/apps/remote_sig.md, line 45 at r1](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/144#-M4Uj_G9G_WCGTsCyGPZ-r1-45:-M4yvu8k3cf0vY3b4ioK:b-lceayv) ([raw file](https://github.com/netsec-ethz/scion-tutorials/blob/1f2a4db3a1e0d694cd0bcfc0db1cb55f0615bb47/content/apps/remote_sig.md#L45)):*
<details><summary><i>Previously, FR4NK-W wrote…</i></summary><blockquote>

>AS=$( ls

Stray space
</blockquote></details>

Space removed. Done.

**Review by slab14 — COMMENTED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/144)* status: all files reviewed, 2 unresolved discussions (waiting on @FR4NK-W)

**Review by FR4NK-W — APPROVED** [review]
> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/144)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**